src/fundus/parser/utility.py

```diff
@@ -177,8 +177,8 @@ def generic_author_parsing(
         return []
 
     parameter_type_error: TypeError = TypeError(
-        f"<value> '{value}' has an unsupported type {type(value)}. "
-        f"Supported types are 'Optional[str], Dict[str, str], List[str], List[Dict[str, str]],'"
+        f"<value> {repr(value)} has an unsupported type {type(value)}."
```

Comment:
That's not necessarily a string so  calling `__repr__` here leads to different results and thus was not intended to be changed with #258.

Reply:
Has been removed.